package/plugin-gargoyle-i18n-English-EN/files/www/i18n/English-EN/email.js

```diff
@@ -15,7 +15,7 @@ email.SmtpSettings="SMTP settings";
 email.ServerIP="Server address:";
 email.ServerPort="Server port:";
 email.Sender="Sender:";
-email.Receipment="Receipment:";
+email.Recipient="Receipment:";
```

Comment:
About english "translation": should be "Recipient:" or "Receipment:"? IMO "Recipient:"

Reply:
Yes, PR updated.